server/cmd/mmctl/commands/post.go

```diff
@@ -103,6 +112,16 @@ func postCreateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func postDeleteCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	postID := args[0]
+
+	if _, err := c.DeletePost(context.TODO(), postID); err != nil {
+		return fmt.Errorf("could not delete post: %s", err.Error())
+	}
```

Comment:
I think there should be a success message similar to the one we have for other commands.